### ISSUE TITLE
"Errno 2"-Msg should get improved

### ISSUE THREAD

**Issue by Tobias-B-Besemer** [bleachbit/bleachbit#439]
I run into the "Errno 2"-Error-Msg... First I thought it's a typo and should be named "Error 2", then after study the source, I saw that this Error is really called this way... Anyway: I got this error, but the error msg doesn't help me to understand what was going wrong, or how I can fix it... I think this error msg should get improved.

![2019-02-12 11_59_46-bleachbit](https://user-images.githubusercontent.com/11881073/52646150-87f55680-2ee2-11e9-98e6-49658d474282.jpg)

**Comment by az0** [comment]
Does "no such file or directory" help you understand what is wrong?

**Comment by Tobias-B-Besemer** [comment]
No, not really! Also: The file exist! But: It seems BleachBit haven't the rights in the file system to delete it!
Therefor a error msg like: "Can't delete file! Insufficient file rights!" would be IMHO better...
I now filled an issue in the wishlist to first change file rights before delete: https://github.com/bleachbit/wishlist/issues/18
